### ISSUE TITLE
TF Keras Save + Load weights/model

### ISSUE THREAD

**Issue by roddtalebi** [ezCGP/ezCGP#76]
Look into how we can load/save weights or the model in keras. Where should we add methods that can do this? Part of EvaluateDefinition for our TF Keras block?

**Comment by roddtalebi** [comment]
One think I always wondered was, "how can i load weights from a model that wasn't created in tensorflow or keras but recreate the architecture as a seed and still set it with pretrained weights?"

looks like tf.keras has a method to set the weights of a layer with a numpy array. so if we can get our hands on the weights somehow and save them as arrays, then we're gold.
https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer#set_weights
